glacier_mapping/train.py

```diff
@@ -40,7 +40,9 @@ def train_epoch(loader, frame, metrics_opts):
       and the metrics on the training set.
     """
     loss, metrics = 0, {}
+    frame.model.train()
     for x, y in loader:
+        frame.optimizer.zero_grad()
```

Comment:
Why is this line necessary? Isn't zero_grad called in frame.optimize()?

Reply:
You're right. I changed it with mode.train and missed that we call it already in frame.optimize()